source/custom/nixnet_converters.h

```diff
@@ -16,6 +16,9 @@ void SetSessionInfoResponse(const u32& input, nixnet_grpc::SessionInfoResponse*
 }  // namespace converters
 
 constexpr auto ENET_FRAME_HEADER_LENGTH = static_cast<u16>(sizeof(nxFrameEnet_t) - 1);  // last byte in nxFrameEnet_t is u8 FrameData[1]
+constexpr auto ENET_FRAME_FCS_SIZE = 4;
+
+u32 get_frame_buffer_size(int32 number_of_frames, u32 max_payload_size, u32 protocol);
```

Comment:
Note : Moved this helper declaration here since I needed to declare this before using it on line 109 below.